test/controllers/items_controller_test.rb

```diff
@@ -0,0 +1,49 @@
+require 'test_helper'
+
+class ItemsControllerTest < ActionController::TestCase
+  setup do
+    @item = items(:one)
+  end
+
+  test "should get index" do
+    get :index
+    assert_response :success
+    assert_not_nil assigns(:items)
+  end
+
+  test "should get new" do
+    get :new
+    assert_response :success
+  end
+
+  test "should create item" do
+    assert_difference('Item.count') do
+      post :create, item: { name: @item.name, price: @item.price }
+    end
+
+    assert_redirected_to item_path(assigns(:item))
+  end
+
+  test "should show item" do
+    get :show, id: @item
+    assert_response :success
+  end
+
+  test "should get edit" do
+    get :edit, id: @item
+    assert_response :success
+  end
+
+  test "should update item" do
+    patch :update, id: @item, item: { name: @item.name, price: @item.price }
+    assert_redirected_to item_path(assigns(:item))
+  end
+
+  test "should destroy item" do
+    assert_difference('Item.count', -1) do
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.